src/samples/ChangeOfBank/ChangeOfBank.tsx

```diff
@@ -32,7 +32,7 @@ export default function ChangeOfBank() {
   const { hmrcURL } = useHMRCExternalLinks();
   const setAuthType = useState('gg')[1];
 
-  registerServiceName(t('CHB_HOMEPAGE_HEADING'));
+  registerServiceName(t('CHILD_BENEFIT'));
```

Comment:
update this same as the service name for COB

Reply:
to the same as the heading? i.e. 
COB_SERVICE_HEADING

Reply:
yes